interface/src/app/gene/searchMany/searchMany.js

```diff
@@ -25,6 +25,15 @@ angular.module('adage.gene.searchMany', [
               // be obtained from the ML model. This is the same as
               // the issue in geneSearchForm (also with $scope.organism).
               $scope.organism = 'Pseudomonas aeruginosa';
+
+              // '$scope.autocomplete.autocomlete' holds a boolean value,
+              // of whether or not the user wants to use autocomplete
+              // search to look up a few genes. If not, then the autocomplete
+              // search panel will disappear, and the panel to search many
+              // genes will appear. We need to send the value itself inside
+              // an object so that it gets sent properly back and forth
+              // through the corresponding $scopes.
+              $scope.autocomplete = {autocomplete: true};
```

Comment:
I did a test and found that the change on a boolean flag that is shared by the parent scope and its directive child scope can be detected correctly. What was the error you got when defining `autocomplete` as a boolean?